src/components/VrtConversion/Withdraw.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Icon } from 'antd';
```

Comment:
what do you think about deprecation of using antd and starting to use v2 components for new components in frontend?

Reply:
make sense, I will add a new `Icon` component in v2